tests/test_integration.py

```diff
@@ -462,23 +462,46 @@ async def test_validate(envsonschema, tmp_path):
 
 @pytest.mark.asyncio
 async def test_summary(envsonschema, tmp_path):
-    tmp_path.joinpath("schema.json").write_text("{}")
-    tmp_path.joinpath("instance.json").write_text("12")
+    tmp_path.joinpath("schema.json").write_text(
```

Comment:
Rather than modifying the existing test (which tests the existing behavior) you should add a second one which tests your new behavior, and leave the first one in place, because we don't want to *stop* testing the old behavior, we want to make sure some new behavior is *also* tested.

In other words, a second test called, say, `test_summary_show_validation`, which you can mostly copy from this test case, add your `--show validation` option as you have, and then likely the rest of the test as you have modified here.

You may need more than one in fact, perhaps 2 or 3, which cover one case where there's no errors or skipped tests, just valid or invalid instances, and then a second test where there's a skipped test and a test that an implementation errored on. This way we can be sure the code written works without testing it manually.